File: test/fixtures/sample.ts
interface User {
  id: number;
  name: string;
  email: string;
}

function greet(user: User): string {
  return `Hello, ${user.name}!`;
}

const user: User = { id: 1, name: "Alice", email: "alice@example.com" };
console.log(greet(user));

File: test/fixtures/sample.py
from dataclasses import dataclass
from typing import List

@dataclass
class User:
    id: int
    name: str
    email: str

def greet(user: User) -> str:
    return f"Hello, {user.name}!"

users: List[User] = [
    User(1, "Alice", "alice@example.com"),
    User(2, "Bob", "bob@example.com"),
]

for user in users:
    print(greet(user))
